### PR TITLE
Feature: Add ask eval run option - use optimal db schema

### PR DESCRIPTION
**Pull request by imAsterSun**
Add a run option to ask evaluation pipeline to use given optimal db schema

### CODE REVIEW

File: wren-ai-service/src/eval/ask/__main__.py [hunk 1]
```diff
@@ -244,6 +256,11 @@ def eval(prediction_results_file: Path, dataset_name: str, ground_truths: list[d
         action=argparse.BooleanOptionalAction,
         help="Whether to use hard questions for evaluation. Default is False.",
     )
+    parser.add_argument(
+        "--use-optimal-schema",
+        action=argparse.BooleanOptionalAction,
+        help="Whether to use hard questions for evaluation. Default is False.",
```
Comment (cyyeh) [inline wren-ai-service/src/eval/ask/__main__.py:262]: help message should be corrected

File: wren-ai-service/src/eval/ask/__main__.py [hunk 2]
```diff
@@ -361,6 +379,9 @@ def eval(prediction_results_file: Path, dataset_name: str, ground_truths: list[d
     if not Path("./outputs").exists():
         Path("./outputs").mkdir()
 
+    with open(f"./src/eval/data/{DATASET_NAME}_optimal_ddl.json", "r") as f:
```
Comment (cyyeh) [inline wren-ai-service/src/eval/ask/__main__.py:382]: you should only read this file only if USE_OPTIMAL_SCHEMA is True

**Review by cyyeh — COMMENTED** [review]
Overall is LGTM

File: wren-ai-service/src/eval/ask/__main__.py (hunk 2 above)
Comment (imAsterSun) [reply wren-ai-service/src/eval/ask/__main__.py]: Thanks for the suggestion, the code is updated.

**Review by cyyeh — APPROVED** [review]
LGTM

**Pull request merged**